PreMailer.Net/PreMailer.Net.Tests/Issue410Tests.cs

```diff
@@ -0,0 +1,25 @@
+using AngleSharp.Html.Parser;
+using Xunit;
+
+namespace PreMailer.Net.Tests
+{
+    public class Issue410Tests
+    {
+        [Fact]
+        public void MoveCssInline_PreservesImportantInInlineStyles()
+        {
+            string input = @"<style> 
+.test {
+ color:red;
+ }
+ </style>
+<body>
+<p class=""test"" style=""font-weight: bold !important;"">test</p>
+</body>";
+
+            var result = PreMailer.MoveCssInline(input);
+
+            Assert.Contains("font-weight: bold !important", result.Html);
+        }
+    }
+}
```

Comment:
Don't create tests with vague naming, such as Issue410 tests - reading that code doesn't convey meaning.

Can this be tested more isolated as part of `StyleClassApplierTests`? If not, move this test to the `PreMailerTests` class and align the naming.